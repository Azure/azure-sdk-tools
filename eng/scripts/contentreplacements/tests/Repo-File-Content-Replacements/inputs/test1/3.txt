3.txt
This is a testing for azsdk-pool-mms-win-2019-general migration. 
This is a testing for windows2019 migration. 
Expect to have changes for this file.
