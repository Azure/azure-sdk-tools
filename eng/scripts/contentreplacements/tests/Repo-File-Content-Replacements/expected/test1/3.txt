3.txt
This is a testing for azsdk-pool-mms-win-2022-general migration. 
This is a testing for windows2022 migration. 
Expect to have changes for this file.
